rfcs/rfc-using-pinia-for-form-components.md

```diff
@@ -0,0 +1,318 @@
+- Start Date: 2023-03-06
+- Target Major Version: n/a
+- Reference Issues: bcgov/entity#15509
+- Entity Issue: 
+- Implementation PR: 
+
+# Summary
+
+This RFC seeks feedback to help improve the application architecture of
+business-filings-ui, business-create-ui and business-filings-ui.  At present,
+we use props and events to communicate between form components like this:
```

Comment:
Good points @cameron-eyds and @thorwolpert the title of this RFC is misleading. I'll change the title and some of the content to make clear what we're proposing is an architectural change that uses global state rather than Pinia or Vuex specifically.